### PR TITLE
Fixed doc adding_non_namespaced_resources with correct steps

### PR DESCRIPTION
**Pull request by raushan2016**
- Steps in the document are out-dated and doesn't work. 
- Updated with the steps which does work.

### CODE REVIEW

**Comment by raushan2016** [comment]
I signed it

File: docs/adding_non_namespaced_resources.md [hunk 1]
```diff
@@ -17,27 +17,31 @@ Use the `--non-namespaced=true` flag when creating a resource:
 
 Non-namespaced resources have the following differences from namespaced resources:
 
-- nonNamespaced comment directive above the type
-  - `// +nonNamespaced=true` comment under `// +genclient=true`
-- Strategy and StatusStrategy override NamespacedScoped to false
+- nonNamespaced comment directive above the type in `pkg/apis/{group}/{version}/{Kind}_types.go`
+  - `// // +genclient:nonNamespaced` comment under `// +genclient`
```
Comment (yue9944882) [inline docs/adding_non_namespaced_resources.md]: duplicated slash

Comment (raushan2016) [reply docs/adding_non_namespaced_resources.md]: fixed

**Pull request merged**